scout/commands/base.py

```diff
@@ -48,7 +49,8 @@
 @click.pass_context
 def cli(context, mongodb, username, password, authdb, host, port, loglevel, config, demo):
     """scout: manage interactions with a scout instance."""
-    coloredlogs.install(level=loglevel)
+    log_format = "%(message)s" if sys.stdout.isatty() else None
+    coloredlogs.install(level=loglevel, fmt=log_format)
```

Comment:
Nice - would be better with colored text for "info" logs to make it stand out but it's work in progress 🙂